examples/examples/get_blocks_async.rs

```diff
@@ -0,0 +1,73 @@
+/*
+	Copyright 2019 Supercomputing Systems AG
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+//! Very simple example that shows how to fetch chain metadata with async.
+
+use kitchensink_runtime::Runtime;
+use sp_core::sr25519;
+use std::time::Instant;
+use substrate_api_client::{
+	ac_primitives::PlainTipExtrinsicParams, rpc::JsonrpseeClient, Api, GetChainInfo,
+};
+
+#[cfg(all(feature = "sync-examples", feature = "async-examples",))]
```

Comment:
The feature shenanigans are a bit annoying... but I couldn't think of a better way to compile the async example (without creating yet another example directory)